### PR TITLE
French PSA folder as an example 

### PR DESCRIPTION
**Pull request by lh1008**
Hi @erciccione,
 
This folder will help translators and see examples of how the translations are being worked. 

@thunderosa @xmrhaelan

### CODE REVIEW

**Comment by erciccione** [comment]
See my comments in [#18 (comment)](https://github.com/monero-ecosystem/outreach-docs/pull/18#pullrequestreview-156981572) for an initial technical review.

**Comment by lh1008** [comment]
Hi @el00ruobuob,

Could you please review this PR?

**Comment by lh1008** [comment]
Did suggested changes, removed folder 'fr' inside 'fr' folder, fonts, and autosave files.

**Comment by el00ruobuob** [comment]
Sure @lh1008, i do it right now.

File: monero-outreach-docs/translations/fr/PSA/files/forks/forks_fr.sla [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="10.5" FSHADE="90" CH="article ou deux sur les hards forks et comment cela a créé des conflits ou des drames entre certains projets cryptographiques. "/>
+            <para LINESP="12.5"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="10.5" FSHADE="90" CH="Un hard fork de Monero se manifeste de manière intéressante "/>
+            <para LINESP="12.5"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="10.5" FSHADE="90" CH="et se divise en trois catégories principales :"/>
+            <para LINESP="12.5"/>
+            <para LINESP="12.5"/>
+            <ITEXT FONT="Adobe Caslon Pro Bold" FONTSIZE="10" FSHADE="90" CH="1) Hard Fork Régulièrement Programmé"/>
+            <para LINESP="12.5"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="10.5" FSHADE="90" CH="Les développeurs de hard fork visent à planifier deux forks "/>
```
Comment (el00ruobuob) [inline monero-outreach-docs/translations/fr/PSA/files/forks/forks_fr.sla]: s/développeurs de hard fork/développeurs de Monero/

File: monero-outreach-docs/translations/fr/PSA/files/forks/forks_fr.sla [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="10" FSHADE="90" CH="."/>
+            <para LINESP="12.5"/>
+            <para LINESP="12.5"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="10.5" FSHADE="90" CH="D’autres projets ont tiré parti des hard forks réguliers de "/>
+            <para LINESP="12.5"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="10.5" FSHADE="90" CH="Monero et ont créé leurs propres projets sur les anciennes "/>
+            <para LINESP="12.5"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="10.5" FSHADE="90" CH="chaînes de Monero. Ces projets échouent souvent en peu de "/>
+            <para LINESP="12.5"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="10.5" FSHADE="90" CH="temps et se nourrissent de la nature gourmande des individus. "/>
```
Comment (el00ruobuob) [inline monero-outreach-docs/translations/fr/PSA/files/forks/forks_fr.sla]: s/de la nature gourmande/de l'avidité/

File: monero-outreach-docs/translations/fr/PSA/files/learning/learning_fr.sla [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            <trail/>
+            <PageItemAttributes/>
+        </PAGEOBJECT>
```
Comment (el00ruobuob) [inline monero-outreach-docs/translations/fr/PSA/files/learning/learning_fr.sla]: No text in here?

File: monero-outreach-docs/translations/fr/PSA/files/teamwork/teamwork_fr.sla [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            <trail/>
+            <PageItemAttributes/>
+        </PAGEOBJECT>
```
Comment (el00ruobuob) [inline monero-outreach-docs/translations/fr/PSA/files/teamwork/teamwork_fr.sla]: No text here?

File: monero-outreach-docs/translations/fr/PSA/files/wallet/wallet_fr.sla [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+            <trail/>
+            <PageItemAttributes/>
+        </PAGEOBJECT>
```
Comment (el00ruobuob) [inline monero-outreach-docs/translations/fr/PSA/files/wallet/wallet_fr.sla]: No text here?

File: monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr-printerfriendly.sla [hunk 6]
```diff
@@ -89,6 +82,22 @@
...
+            <ITEXT FONT="Adobe Caslon Pro Bold" FONTSIZE="12" CH="Laboratoire de Recherche Monero :"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="12" CH=" Monero n'est pas uniquement dévoué à créer une cryptomonnaie fongible, mais aussi à continuer la recherche dans le domaine de la confidentialité financière dans la mesure où cela fait appel aux cryptomonnaies d'une manière générale. Dans ce but, des chercheurs en Mathématique et Physique computationnelle ont publiés "/>
```
Comment (el00ruobuob) [inline monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr-printerfriendly.sla]: s/dévoué/voué/

File: monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr-printerfriendly.sla [hunk 7]
```diff
@@ -1689,57 +1582,156 @@
...
+            <ITEXT CH="s, et sans frontières, les gens peuvent envoyer facilement de l'argent indépendamment de la corruption ou de la défaillance des gouvernements ou des banques.  Cela permet une prise de pouvoir économique des individus dans des "/>
+            <ITEXT FONT="Adobe Caslon Pro Semibold" FONTSIZE="12" FEATURES="inherit underline" FCOLOR="quickfacts" CH="pays oppresseurs"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="12" CH=" ou des économies en crise."/>
+            <para LINESP="13.5"/>
+            <para LINESP="13.5"/>
+            <ITEXT CH="L'historique financier confidentiel protège les consommateurs et les compagnies des manipulations de prix, l'exploitation de la chaîne d'approvisionnement, la discrimination économique, ou autres."/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="12" CH=" "/>
+            <ITEXT FONT="Adobe Caslon Pro Semibold" FONTSIZE="12" FEATURES="inherit underline" FCOLOR="quickfacts" CH="Monero est la seule cryptomonnaie"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="12" CH=" qui possède les caractéristiques d'une monnaie électronique entièrement fongibles et décentralisée."/>
```
Comment (el00ruobuob) [inline monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr-printerfriendly.sla]: s/fongibles/fongible/

File: monero-outreach-docs/translations/fr/PSA/headers/PSA _fr_headers.txt [hunk 8]
```diff
@@ -0,0 +1,49 @@
+﻿ Page 1
+- My Monero is tight
+mon monero est á l'abri
```
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/headers/PSA _fr_headers.txt]: Done. I'm working on these changes. My keyboard is in spanish and english, I will have to add more language keyboards. :)

File: monero-outreach-docs/translations/fr/PSA/headers/PSA _fr_headers.txt [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+ - Monero Hero says:
+l'héroïne monero dit:
+
+ - Don’t be a Jive Turkey
+fais pas le con
+
+
+Page 3 
+ - there are 52 learning weeks in a year
+il y a 52 semaines d'apprentissage dans une anée
```
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/headers/PSA _fr_headers.txt]: done.

File: monero-outreach-docs/translations/fr/PSA/headers/PSA _fr_headers.txt [hunk 10]
```diff
@@ -0,0 +1,49 @@
...
+
+
+Page 4
+ - Shut it To The max
+tu vas la fermer ta grande geule !?
```
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/headers/PSA _fr_headers.txt]: done.

File: monero-outreach-docs/translations/fr/PSA/headers/PSA _fr_headers.txt [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+
+
+Page 5
+ - Monero Teamwork
+le travail d'équipe monero
+
+-from start to finish!
+-du début á la fin!
```
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/headers/PSA _fr_headers.txt]: done.

File: monero-outreach-docs/translations/fr/PSA/headers/PSA _fr_headers.txt [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+
+
+Page 6
+ - How about we go back to my place and you give me your xmr private keys?
+alors, on retourne chez moi et tu me donnes tes clés privées XMR?
```
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/headers/PSA _fr_headers.txt]: done.

File: monero-outreach-docs/translations/fr/PSA/headers/PSA _fr_headers.txt [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+
+ - Keep on dreaming, lover boy…
+continue á rêver, mon chou
```
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/headers/PSA _fr_headers.txt]: done.

File: monero-outreach-docs/translations/fr/PSA/headers/PSA _fr_headers.txt [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+
+ - ...keep on dreaming
+continue a rêver
```
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/headers/PSA _fr_headers.txt]: done.

File: monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr-printerfriendly.sla (hunk 6 above)
Comment (lh1008) [reply monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr-printerfriendly.sla]: done, changed.

File: monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr-printerfriendly.sla (hunk 7 above)
Comment (lh1008) [reply monero-outreach-docs/translations/fr/quickfacts/QuickFacts_fr-printerfriendly.sla]: done

File: monero-outreach-docs/translations/fr/PSA/files/dontshare/dontshare_fr.sla [hunk 15]
```diff
@@ -0,0 +1,103 @@
...
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" CH="Vos Monero sont en sûreté tant que vos clés privées et vos phrases mnémoniques privées sont en sécurité. Vos clés privées et vos phrases mnémoniques privées sont les seuls outils vous permettant d’accéder à votre portefeuille et de dépenser / envoyer vos XMR. Si vous les partagez, alors ceux avec qui vous les partagez (ou ceux avec qui vous les partagez qui ont l’expertise technique pour les voler) ont autant accès à votre portefeuille que vous."/>
```
[the remark this answers is not in the document]
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/PSA/files/dontshare/dontshare_fr.sla]: Sorry, i should have re-read the original version.
I suggest to change "ou ceux avec qui vous les partagez qui ont l’expertise technique pour les voler" to "ou ceux qui ont l’expertise technique pour les voler à ceux avec qui vous les partagez"

File: monero-outreach-docs/translations/fr/PSA/files/learning/learning_fr.sla (hunk 3 above)
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/PSA/files/learning/learning_fr.sla]: NP.

File: monero-outreach-docs/translations/fr/PSA/files/teamwork/teamwork_fr.sla (hunk 4 above)
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/PSA/files/teamwork/teamwork_fr.sla]: NP.

File: monero-outreach-docs/translations/fr/PSA/files/wallet/wallet_fr.sla (hunk 5 above)
Comment (el00ruobuob) [reply monero-outreach-docs/translations/fr/PSA/files/wallet/wallet_fr.sla]: NP.

File: monero-outreach-docs/translations/fr/PSA/files/dontshare/dontshare_fr.sla (hunk 15 above)
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/files/dontshare/dontshare_fr.sla]: ok done.

File: monero-outreach-docs/translations/fr/PSA/files/dontshare/dontshare_fr.sla [hunk 16]
```diff
@@ -0,0 +1,103 @@
...
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" CH="Parfois, les autres projets de crypto-monnaie vont faire un hard fork du code Monero et clameront que vous recevez 10 x le montant de vos pièces et que «tout ce "/>
```
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/files/dontshare/dontshare_fr.sla]: done

File: monero-outreach-docs/translations/fr/PSA/files/dontshare/dontshare_fr.sla [hunk 17]
```diff
@@ -0,0 +1,103 @@
...
+            <para LINESP="13.2"/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" CH="Vous pensez peut-être que vous faites une bonne "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" CH="chose en gardant vos clés privées séparées de votre emplacement physique, mais que se passera-t-il si grand-mère oublie vos instructions, les laisse sur le rebord de "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" CH="la fenêtre et qu’un passant les voit et les note? Vous vous réveillerez le lendemain matin et tous vos XMR auront été envoyés à une adresse inconnue. Étant donné que Monero est ultra privé, vous n’aurez aucun moyen de savoir à qui appartient le porte-monnaie inconnu et vous ne pourrez pas récupérer vos XMR manquants ou volés."/>
```
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/files/dontshare/dontshare_fr.sla]: done

File: monero-outreach-docs/translations/fr/PSA/files/dontshare/dontshare_fr.sla [hunk 18]
```diff
@@ -0,0 +1,103 @@
...
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" CH="Parfois, les autres projets de crypto-monnaie vont faire un hard fork du code Monero et clameront que vous recevez 10 x le montant de vos pièces et que «tout ce "/>
+            <para LINESP="13.2"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" CH="que vous avez à faire pour réclamer ce montant de "/>
+            <para LINESP="13.2"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" CH="pièces est de partager vos clés privées». Si vous apercevez ce genre d’offre, n’y touchez pas, évitez-la comme la peste. Non seulement les arnaqueurs derrière cette offre tentent de vous piéger, mais ils peuvent aussi essayer de compromettre l’intégrité de Monero dans son ensemble. Les arnaqueurs dépendent de votre avidité pour commettre beaucoup d’actions dommageables."/>
```
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/files/dontshare/dontshare_fr.sla]: done

File: monero-outreach-docs/translations/fr/PSA/files/donttalk/donttalk_fr.sla [hunk 19]
```diff
@@ -0,0 +1,115 @@
...
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="en cryptomonnaies, il serait préférable qu’il n’ait pas connaissance de vos Monero. Même si le code Monero résiste aux contrôles de capitaux, si un gouvernement abusif sait que vous possédez des Monero, il peut quand même venir frapper à votre porte."/>
+            <para/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="Les contrôles de capitaux leur donneraient techniquement l’autorisation légale de geler ou "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="de prendre possession de vos Monero. Au fil de l’amélioration de Monero, il deviendra de plus en plus difficile pour les gouvernements abusifs de savoir que vous en possédez.... à moins d’en parler en ligne. S’ils ne savent pas que vous en possédez, leur capacité d’abuser de leur pouvoir et de voler votre patrimoine durement gagnée s’amoindrit. Ne racontez pas combien de Monero vous avez."/>
```
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/files/donttalk/donttalk_fr.sla]: done

File: monero-outreach-docs/translations/fr/PSA/files/exchange/exchange_fr.sla [hunk 20]
```diff
@@ -0,0 +1,150 @@
...
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="10.7" FSHADE="90" KERN="-0.3" CH="Les bourses d’échange peuvent être une manière pratique d’acheter et de vendre plus de Monero, mais toutes les bourses "/>
+            <para LINESP="11.5"/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="10.7" FSHADE="90" KERN="-0.3" CH="ne naissent pas égales."/>
+            <para LINESP="11.5"/>
+            <para LINESP="11.5"/>
+            <ITEXT FONT="Adobe Caslon Pro Bold" FONTSIZE="10.7" FSHADE="90" KERN="-0.3" CH="Voici quelques éléments à garder en tête lorsque vous "/>
```
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/files/exchange/exchange_fr.sla]: done

File: monero-outreach-docs/translations/fr/PSA/files/exchange/exchange_fr.sla [hunk 21]
```diff
@@ -0,0 +1,150 @@
...
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="10.6" FSHADE="90" CH="d. Les créateurs / gestionaires sont-ils individus ou entités connus ou réputés ?"/>
```
[the remark this answers is not in the document]
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/files/exchange/exchange_fr.sla]: done

File: monero-outreach-docs/translations/fr/PSA/files/learning/learning_fr.sla (hunk 3 above)
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/files/learning/learning_fr.sla]: added

File: monero-outreach-docs/translations/fr/PSA/files/teamwork/teamwork_fr.sla (hunk 4 above)
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/files/teamwork/teamwork_fr.sla]: added

File: monero-outreach-docs/translations/fr/PSA/files/wallet/wallet_fr.sla (hunk 5 above)
Comment (lh1008) [reply monero-outreach-docs/translations/fr/PSA/files/wallet/wallet_fr.sla]: added

**Comment by erciccione** [comment]
@lh1008 Contact me on a live chat, i can help you to fix those :)

**Comment by lh1008** [comment]
OK guys,

@el00ruobuob and @erciccione the suggested changes were made. Just need a final view and approval so we can merge it to the master branch. 

Thank you guys for your help.

**Pull request merged**